docker-cloud/getting-started/connect-infra.md

```diff
@@ -11,7 +11,7 @@ To deploy Docker Cloud nodes, you first need to grant Docker Cloud access to you
 This could mean granting access to a cloud services provider such as AWS or Azure, or installing the Docker Cloud Agent on your own hosts. Once this is done, you can provision nodes directly from within Docker Cloud using the Web UI, CLI or API.
 
 ## Link to a cloud service provider
-To link your cloud provider accounts, first go to your [Docker Cloud account dashboard](https://cloud.docker.com/account/). (You can also click your username in the top right corner and choose **Account info**.)
+To link your cloud provider accounts, first go to your [Docker Cloud dashboard](https://cloud.docker.com/). If you are not already logged in, choose **Login with Docker ID** in the top right corner to do so. (On the dashboard, you'll find your username, account settings, and other options in the top right.)
```

Comment:
I would actually prefer not to give login instructions here. If you put them here, you end up with them on every procedure. ;) 
I feel like this is signaled well enough from the UI that if they're interrupted by the sign in screen, they should be able to figure this out on their own.